src/python/pants/backend/kotlin/compile/kotlinc_test.py

```diff
@@ -365,14 +392,24 @@ def test_compile_with_undeclared_jvm_artifact_dependency_fails(
     assert "unresolved reference: joda" in fallible_result.stderr
 
 
+@pytest.fixture
+def kotlinc_allopen_lockfile_def() -> JVMLockfileFixtureDefinition:
+    return JVMLockfileFixtureDefinition(
+        "kotlinc-allopen.test.lock",
+        [f"org.jetbrains.kotlin:kotlin-allopen:{_KOTLIN_VERSION}"] + KOTLIN_STDLIB_REQUIREMENTS,
+    )
+
+
+@pytest.fixture
+def kotlinc_allopen_lockfile(
+    kotlinc_allopen_lockfile_def: JVMLockfileFixtureDefinition, request
```

Comment:
So `request` is a magical fixture parameter name I guess?

Reply:
Yes, the `request` fixture is the metadata fixture that allows tests to introspect themselves.